components/PicturesListContainer.js

```diff
@@ -1,74 +1,63 @@
 import React, {Component} from 'react';
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+
+import fetchDataAction from '../actions/fetchData';
 import PicturesListComponent from './PicturesListComponent';
-import {Linking} from 'react-native';
+
 
 class PicturesListContainer extends Component {
   constructor(props) {
     super(props);
-
-    this.state = {
-      isLoading: false,
-      data: [],
-      error: null,
-    };
   }
 
   componentDidMount() {
-    this.fetchData();
+    const {fetchData} = this.props;
+    fetchData();
   }
 
-  fetchData = () => {
-    const url = 'https://picsum.photos/v2/list';
-    this.setState({
-      isLoading: true,
-    });
-    fetch(url)
-      .then(response => response.json())
-      .then(response => {
-        this.setState({
-          data: response,
-          isLoading: false,
-        });
-      })
-      .catch(() => {
-        this.setState({
-          error: true,
-        });
-      });
-  };
-
   sortDataByAuthor = () => {
```

Comment:
Create one method to sort - with arg that will define if this is an author or Id sort